src/veh_type.h

```diff
@@ -89,6 +89,12 @@ class vpart_info
         /** What slot of the vehicle tile does this part occupy? */
         std::string location;
 
+        /**
+         * What other locations are provided for mounting of other parts
+         * first is max volume, second max weight for all mounted parts
+         */
+        std::map<std::string, std::pair<int,int>> mounts;
```

Comment:
A simple struct containing `volume` and `weight` members instead of a pair (which provides `first` and `second`) would be better.

Seeing `first` and `second` all over the place where nobody knows what they mean reduces the readability a lot.